### DISCUSSION TITLE
Draw Using Palette Order for Geometry Tool

### DISCUSSION THREAD

**Discussion by KaseyFarron** [opentoonz/opentoonz#6826] [Ideas]
**Is your feature request related to a problem? Please describe**
Drawing using the _Palette Order_ found under brush settings on a Toonz Raster level is not available when using the Geometry Tool.

**Describe the solution you'd like**
For it being possible to use the palette order for drawing behind, in front, etc.

**Comment by MikeTysonsPidgeon** [comment]
I second this. Also, having the drop down that allows for selecting whether you use pallet order or simply draw under/over. I initially wanted to switch over to the geometry tools for clean up, but I got frustrated with the lack of the draw order control that the brush provided.

**Comment by MikeTysonsPidgeon** [comment]
This is still something I'm interested in.
